src/Symfony/Component/Messenger/Asynchronous/Routing/SenderLocatorInterface.php

```diff
@@ -25,7 +26,16 @@
      *
      * @param object $message
      *
-     * @return SenderInterface[]
+     * @return SenderInterface|null
      */
-    public function getSendersForMessage($message): array;
+    public function getSenderForMessage($message): ?SenderInterface;
+
+    /**
+     * Whether to route the message to a sender and still have it passed to its respective handler.
+     *
+     * @param object $message
+     *
+     * @return bool
+     */
+    public function forwardToSenderAndHandler($message): bool;
```

Comment:
Isn't this method name a bit misleading? I didn't expected it to be as "isser" at first, but something performing an action.
Shouldn't it be something like `mustForwardToSenderAndHandler` (or better)?

Reply:
Yeah makes sense to change it.